lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -157,14 +157,25 @@ SWAP_BP_FOR_OPCODE:
                 return nullptr;
 
             case INTERPRETER_OPCODE::ExtendedOpcodePrefix:
+            case INTERPRETER_OPCODE::DblExtendedOpcodePrefix:
```

Comment:
Discussing offline with @curtisman, he made a good point that this should have its own case arm.